### ISSUE TITLE
Compiler error with Crystal 0.21.1

### ISSUE THREAD

**Issue by sepulworld** [mrrooijen/commander#11]
Thanks for sharing this shard, good stuff!

I found some compiler errors when using this shard with Crystal 0.21.1

```
$ crystal --version
Crystal 0.21.1 (2017-03-14) LLVM 4.0.0
```

Testing with the provided example in the README.MD and getting following compile error

```
Module validation failed: inlinable function call in a function with debug info must have a !dbg location
  %74 = call i8* @__crystal_malloc(i32 ptrtoint (%"Commander::Command"* getelementptr (%"Commander::Command", %"Commander::Command"* null, i32 1) to i32))

0x1018d0b53: *raise<String>:NoReturn at ??
0x1028a1aab: *Crystal::CodeGenVisitor#finish:Nil at ??
0x1027cd37e: *Crystal::Compiler#codegen<Crystal::Program, Crystal::ASTNode+, Array(Crystal::Compiler::Source), String>:(Tuple(Array(Crystal::Compiler::CompilationUnit), Array(String)) | Nil) at ??
0x103072ff4: *Crystal::Compiler#compile<Array(Crystal::Compiler::Source), String>:Crystal::Compiler::Result at ??
0x10192f1eb: *Crystal::Command#run:(Bool | Crystal::Compiler::Result | IO::FileDescriptor | Nil) at ??
0x1018fa8dd: main at ??
```

**Comment by N0r1x** [comment]
I'm using the following debug version 
Crystal 0.21.1+33 [917971ded] (2017-03-26) LLVM 3.9.1
and the test program from the README compiles here
Btw: there's a command line interface thing in the standard library now: 
https://crystal-lang.org/api/0.18.7/OptionParser.html

**Comment by mrrooijen** [comment]
Appears to be fixed in [Crystal 0.22.0](https://github.com/crystal-lang/crystal/releases/tag/0.22.0).
